WebClient/Scripts/webSocket.js

```diff
@@ -78,7 +91,9 @@ var wsLookup = {
         detailsControl.resetkpi(payload); //todo implement KPI's?
     },
     updatechart: function (payload) {
-        GraphManager.UpdateGraphs(payload);
+        if (graphService) {
+            graphService.updateGraphs(payload);
+        }
     },
     showchart: function (payload) {

```

Comment:
showelement could be called when clicking a label on the chart to show a "child" chart